src/OSFS.h

```diff
@@ -42,6 +42,8 @@
 #include <Arduino.h>
 
 namespace OSFS {
+	//File name lengths
+	constexpr uint8_t fileName_length = 11;
```

Comment:
The convention in the rest of the code is UPPERCASE_NAMES for constant variables. Could you change to this? Also, I think a `size_t` is a more appropriate type here.